src/core/models/api-urls.ts

```diff
@@ -0,0 +1,7 @@
+import axios from "axios"
+
+const apiUrl = 'http://localhost:8081/api'
+const apiLoginUrl = apiUrl + '/user/login'
+const apiSignupUrl = apiUrl + '/user/sign-up'
+
+export default { apiUrl, apiLoginUrl, apiSignupUrl}
```

Comment:
This isn't a model (based on directory path and content of this file) so this does not belong here.